packages/@romejs/compiler/lint/rules/js/unusedVariables.test.ts

```diff
@@ -0,0 +1,37 @@
+import {test} from "rome";
+import {testLintMultiple} from "../testHelpers";
+import {dedent} from "@romejs/string-utils";
+
+test(
+	"enforce to not declare variables that are not used",
+	async (t) => {
+		await testLintMultiple(
+			t,
+			[
+				// INVALID
+				`const a = 4;`,
+				`let a = 4;`,
+				dedent`function foo(myVar) { 
```

Comment:
You should indent is this way otherwise it won't work.
```suggestion
				dedent`
				  function foo(myVar) { 
```